app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,19 @@
 <!DOCTYPE html>
 <html lang="<%= I18n.locale %>">
 	<head>
-		<title>Bedpost</title>
+		<% unless analytics_id.nil? %>
+			<!-- Global site tag (gtag.js) - Google Analytics -->
+			<script async src="https://www.googletagmanager.com/gtag/js?id=<%= analytics_id %>"></script>
+		<% end %>
+
+		<%= javascript_tag nonce: true do%>
+			window.timing = Date.now();
```

Comment:
`performance.now()` is apparently more accurate
https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
it doesn't reset on turbolinks page loads, but I guess I could use marks...